File: Python/520A. Pangram.py
def main():
    n = int(input())
    s = list(input())
    a = 97
    i = 0
    count = 0
    while i<n:
        if s[i].isupper():
            s[i] = s[i].lower()
        if ord(s[i]) == a:
            count += 1
            a += 1
            i = -1
        if count == 26:
            break
        i += 1
    if count==26:
        print("YES\n")
    else:
        print("NO\n")
main()

File: C++/520A. Pangram.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n, count=0;
    string s;
    cin>>n;
    cin>>s;
    int a=97;
    for(int i=0; i<n; i++){
        if(isupper(s[i])){
            s[i] = tolower(s[i]);
        }
        if(int(s[i])==a){
            count++;
            a++;
            i = -1;
        }
        if(count==26){
            break;
        }
    }
    if(count==26){
        cout<<"YES"<<endl;
    }
    else{
        cout<<"NO"<<endl;
    }
    return 0;
}